addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -185,21 +187,35 @@ wTourUtils.registerWebsitePreviewTour('edit_link_popover', {
         run: function () {}, // it's a check
     },
     {
-        content: "Ensure that a click on the link popover link opens a new window",
+        content: "Ensure that a click on the link popover link opens a new window in edit mode",
         trigger: 'iframe .o_edit_menu_popover a.o_we_url_link[target="_blank"]',
         extra_trigger: 'iframe .o_edit_menu_popover a.o_we_full_url[target="_blank"]',
-        run: function (actions) {
-            this.$anchor[0].addEventListener('click', (ev) => {
-                const originalPreventDefault = ev.preventDefault.bind(ev);
-                ev.preventDefault = () => {
-                    throw new Error(
-                        "The link popover should not be default prevented as to open a new tab"
-                    );
-                };
-                originalPreventDefault(); // We do not want to open a new tab in a tour
-            }, {once: true, capture: true});
-            actions.click();
+        run: async function (actions) {
+            let resolvePromise;
+            patch(browser, 'window_open_action', {
+                open: (url) => {
+                    if (window.location.hostname === url.hostname && url.pathname.startsWith('/@/')) {
+                        let node = document.querySelector('body');
+                        node.classList.add('new_backend_window_opened');
+                        resolvePromise('The link popover will open link in backend');
+                    }
+                }
+            }, { pure: true }) // We do not want to open a link in a tour
+            await new Promise(async (resolve, reject) => {
+                resolvePromise = resolve;
+                await actions.click();
+                reject("The link is not correct or patch failed");
```

Comment:
if the patch fail, the class won't be added, and the next step will fail, this is not needed IMO